crates/nutype_enum/src/lib.rs

```diff
@@ -47,6 +75,7 @@ macro_rules! nutype_enum {
             $(
                 $(#[$variant_attr])*
                 #[allow(non_upper_case_globals)]
+                #[allow(missing_docs)]
```

Comment:
This change is wrong. We should still force variants to require docs if the crate has a requirement for docs